sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -1195,3 +1195,27 @@ async def test_async_queue_cancel_scheduled_messages(self, servicebus_namespace,
 
             messages = await receiver.fetch_next(timeout=120)
             assert len(messages) == 0
+
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
+    @CachedResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    @ServiceBusQueuePreparer(name_prefix='servicebustest', dead_lettering_on_message_expiration=True)
+    async def test_github_uamqp_issue_135(self, servicebus_namespace, servicebus_namespace_key_name,
```

Comment:
Nit:  Let's name the test to be what it validates, and put the github issue it references in a comment.  (One shouldn't need to go to an external site to grok what a test is evaluating :P)